src/main/app/case/definition.ts

```diff
@@ -1484,4 +1484,46 @@ export interface C100OrderTypeInterface {
 
 export interface OtherProceedings {
   order?: C100OrderTypeInterface
-}
\ No newline at end of file
+}
+
+export enum C1AAbuseTypes {
+  PHYSICAL_ABUSE = 'physicalAbuse',
+  PSYCHOLOGICAL_ABUSE = 'psychologicalAbuse',
+  EMOTIONAL_ABUSE = 'emotionalAbuse',
+  SEXUAL_ABUSE = 'sexualAbuse',
+  FINANCIAL_ABUSE = 'financialAbuse',
+  ABDUCTION = 'abduction',
+  WITNESSING_DOMESTIC_ABUSE='witnessingDomesticAbuse',
+  SOMETHING_ELSE='somethingElse',
+}
+
+export enum C1ASafteyConcernsAbout{
+  CHILDREN = 'children',
+  APPLICANT = 'applicant',
+}
+
+export interface C1ASafteyConcernsAbuse{
+  behaviourDetails?: string;
```

Comment:
Should we also store "Which children are you concerned about?"

Reply:
This will be done later when we pick up the story for including child details.